conan/profile_windows_x86_64.txt

```diff
@@ -0,0 +1,16 @@
+[settings]
+os=Windows
+arch=x86_64
+compiler=gcc
```

Comment:
You can GCC to compile on windows???

Reply:
Of course: MinGW is GCC for Windows. And I don't want to messup with MSVC compiler since we heavily rely on some not that really standards features (likes bit fields).

Reply:
Ah good to know, I always though MinGW and GCC were 2 different things. Of course passing by GCC simplifies compatibility.